controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -174,24 +194,105 @@ func (r *StorageConsumerReconciler) reconcilePhases() (reconcile.Result, error)
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageConsumerReconciler) reconcileCephRadosNamespace(
+	isControllerOwner bool,
+	storageCluster *ocsv1.StorageCluster,
+	consumerResources util.StorageConsumerResources,
+) error {
+	blockPools := &rookCephv1.CephBlockPoolList{}
+	if err := r.List(r.ctx, blockPools, client.InNamespace(r.namespace)); err != nil {
+		return err
+	}
+
+	// ensure for this consumer a rados namespace is created in every blockpool
+	for idx := range blockPools.Items {
+		bp := &blockPools.Items[idx]
+		if bp.Name == "builtin-mgr" {
+			continue
+		}
+
+		rns := &rookCephv1.CephBlockPoolRadosNamespace{}
+		rns.Name = fmt.Sprintf("%s-%s", bp.Name, consumerResources.GetRbdRadosNamespaceName())
+		rns.Namespace = r.namespace
+
+		if _, err := ctrl.CreateOrUpdate(r.ctx, r.Client, rns, func() error {
+			if isControllerOwner {
+				if err := controllerutil.SetControllerReference(r.storageConsumer, rns, r.Scheme); err != nil {
+					return err
+				}
+			} else {
+				if err := controllerutil.SetOwnerReference(r.storageConsumer, rns, r.Scheme); err != nil {
+					return err
+				}
+			}
+			rns.Spec.Name = consumerResources.GetRbdRadosNamespaceName()
+			return nil
+		}); err != nil {
+			return err
+		}
+	}
+
+	//TODO: Create the cephClients
+	return nil
+}
+
+func (r *StorageConsumerReconciler) reconcileCephFilesystemSubVolumeGroup(
+	isControllerOwner bool,
+	storageCluster *ocsv1.StorageCluster,
+	consumerResources util.StorageConsumerResources,
```

Comment:
If we only need the subvolumegroup name we should only send the subvolumegroup name